student/models.py

```diff
@@ -8,6 +8,21 @@ class EducationalInstitution(models.Model):
     def __str__(self):
         return self.name
 
+    """
+    Gets all the institutions from the DB, ordered by their name
+    """
+
+    @classmethod
+    def get_institutions(cls):
```

Comment:
This function name does not indicate the business/application use of this function - do not write funcitons because you think you're going to need them, have a concrete plan about where and why you are going to use them and name them accordingly.

Are you going to have a screen that will just list all institutions indiscriminately?

Reply:
I think maybe I'll change it to only return institutions that have enrolled students, this way when recruiters want to filter the students they know which institutions they can choose from

Reply:
is there some question there that I need to answer somehow?